src/lib/OpenEXR/ImfStandardAttributes.h

```diff
@@ -121,6 +122,20 @@ IMF_STD_ATTRIBUTE_DEF (worldToCamera, WorldToCamera, IMATH_NAMESPACE::M44f)
 
 IMF_STD_ATTRIBUTE_DEF (worldToNDC, WorldToNDC, IMATH_NAMESPACE::M44f)
 
+//
+// framelines -- carry one or more JSON-encoded strings each embodying a
+// system for carriage of framing information, relating what was seen in
+// the camera viewfinder to how downstream processing is intended to use
+// the full captured image or some subset thereof.
+//
+// The top-level object represented by this attribute has names calling
+// out the relevant scheme and values carrying that scheme's configuration
+// for this frame. The names should be globally unique and represent the
+// organization that created the associated frameline scheme, e.g. 
+// com.theasc or de.arri.
+
```

Comment:
The schema for ASC FDL can be found here:
https://github.com/ascmitc/fdl/blob/main/FDL_Validation_Tooling/Python_FDL_Checker

Here (as an example of a camera-vendor-specific schema) is ARRI's frame line schema:
<img width="813" alt="image" src="https://user-images.githubusercontent.com/1134845/232874019-b4eaf9d5-7ebc-427f-b409-7ef01d720ce4.png">

(Well, that's part of it. I will try and upload the rest when I have a better network connection than this one through my phone).

Reply:
The schema for ASC FDL can be found here:
https://github.com/ascmitc/fdl/blob/main/FDL_Validation_Tooling/Python_FDL_Checker

Here (as an example of a camera-vendor-specific schema) is ARRI's frame line schema:
<img width="813" alt="image" src="https://user-images.githubusercontent.com/1134845/232874019-b4eaf9d5-7ebc-427f-b409-7ef01d720ce4.png">

(Well, that's part of it. I will try and upload the rest when I have a better network connection than this one through my phone).

Reply:
I think it would be sufficient to put in the comments that the ASC FDL schema is an example of a typical structure expected to be found here. I think we can leave a google search for the URL as an exercise for the reader